CHANGELOG.md

```diff
@@ -4,6 +4,33 @@ All notable changes to this project will be documented in this file.
 Each new release typically also includes the latest modulesync defaults.
 These should not affect the functionality of the module.
 
+## [v9.0.0](https://github.com/voxpupuli/puppet-grafana/tree/v9.0.0) (2021-08-16)
+
+[Full Changelog](https://github.com/voxpupuli/puppet-grafana/compare/v8.0.0...v9.0.0)
+
+**Breaking changes:**
+
+- Support only 6.x/7.x/8.x versions [\#246](https://github.com/voxpupuli/puppet-grafana/pull/246) ([root-expert](https://github.com/root-expert))
+
+**Implemented enhancements:**
+
+- puppetlabs/stdlib: Allow 7.x [\#244](https://github.com/voxpupuli/puppet-grafana/pull/244) ([bastelfreak](https://github.com/bastelfreak))
+- puppet/archive: allow 5.x [\#243](https://github.com/voxpupuli/puppet-grafana/pull/243) ([bastelfreak](https://github.com/bastelfreak))
+- Allow grafana\_team home dashboard to be scoped to a folder [\#241](https://github.com/voxpupuli/puppet-grafana/pull/241) ([treydock](https://github.com/treydock))
+
+**Closed issues:**
+
+- config class does not restart service [\#239](https://github.com/voxpupuli/puppet-grafana/issues/239)
+- grafana\_team resource gives Could not evaluate: Invalid parameter  [\#237](https://github.com/voxpupuli/puppet-grafana/issues/237)
+
+**Merged pull requests:**
+
+- Update README.md about tested OSes [\#250](https://github.com/voxpupuli/puppet-grafana/pull/250) ([bastelfreak](https://github.com/bastelfreak))
+- Use Iterable.find to find a folder [\#249](https://github.com/voxpupuli/puppet-grafana/pull/249) ([ekohl](https://github.com/ekohl))
+- Add Ubuntu 20.04 support [\#248](https://github.com/voxpupuli/puppet-grafana/pull/248) ([root-expert](https://github.com/root-expert))
```

Comment:
Enhancement?

Reply:
Probably no, since only metadata.json was changed

Reply:
We usually do label those as enhancements. As a user it's a nice new feature. In this case it doesn't really matter that much either since it's a major version anyway.

Reply:
regenerated the changelog